WIP-enrollment-CIP.md

```diff
@@ -0,0 +1,9 @@
+In my current estimation, compared to the Plutus- and Ledger-enrolling CIPs above, any CIP enrolling Consensus would be trivial.
```

Comment:
@perturbing said https://github.com/IntersectMBO/ouroboros-consensus/pull/1559#issuecomment-2963187656:

> > any CIP enrolling Consensus would be trivial.

> Can you expand on this statement?

By "it would be trivial" I mean it would just be the quoted paragraphs below, which is the slightest fraction of the Plutus and Ledger enrollment CIPs.